### ISSUE TITLE
SAML dynamic provider sample

### ISSUE THREAD

**Issue by brockallen** [DuendeSoftware/Samples#40]

**Comment by mtinnes** [comment]
Also, looking for this +1

**Comment by mtinnes** [comment]
We are investigating Duende for our IdP but wondering if BFF is compatible with SAML dynamic providers?

**Comment by brockallen** [comment]
> We are investigating Duende for our IdP but wondering if BFF is compatible with SAML dynamic providers?

Those are unrelated. BFF is a pattern you'd use in your client app, and dynamic providers is a feature at IdentityServer to loading the external IdP data dynamically. We only have first class support for OIDC, but our design is extensible and would allow you to store other types of providers into our store and use our dynamic loading system. We have a WS-Fed sample for this already:

https://docs.duendesoftware.com/identityserver/v6/samples/ui/#adding-other-protocol-types-to-dynamic-providers

**Comment by mtinnes** [comment]
Thanks, just starting to wrap my head around this. Our primary requirement is to provide our clients the ability to configure their own SAML login provider for logging into our SAAS. Assuming dynamic providers would meet this req?

**Comment by brockallen** [comment]
Yes, should. I also know @AndersAbel wants to have a sample using his [Sustainsys.Saml2 implementation](https://saml2.sustainsys.com/en/stable/).

**Comment by maartenba** [comment]
Closing this one for now.
